frontend/src/views/RegistrationBox.jsx

```diff
@@ -18,108 +17,189 @@ import CustomInput from "components/CustomInput/CustomInput.jsx";
-                          endAdornment: (
-                            <InputAdornment position="end">
-                              <AccountCircle className={classes.inputIconsColor} />
-                            </InputAdornment>
-                          )
-                        }}
-                      />
-                      <CustomInput
-                        labelText="Email"
-                        id="email"
-                        formControlProps={{
-                          fullWidth: true
-                        }}
-                        inputProps={{
-                          type: "email",
-                          endAdornment: (
-                            <InputAdornment position="end">
-                              <Email className={classes.inputIconsColor} />
-                            </InputAdornment>
-                          )
-                        }}
-                      />
-                      <CustomInput
-                        labelText="Password"
-                        id="pass"
-                        formControlProps={{
-                          fullWidth: true
-                        }}
-                        inputProps={{
-                          type: "password",
-                          endAdornment: (
-                            <InputAdornment position="end">
-                              <LockOutlined className={classes.inputIconsColor} />
-                            </InputAdornment>
-                          )
-                        }}
-                      />
-                      <CustomInput
-                        labelText="Confirm Password"
-                        id="passConfirm"
-                        formControlProps={{
-                          fullWidth: true
-                        }}
-                        inputProps={{
-                          type: "password",
-                          endAdornment: (
-                            <InputAdornment position="end">
-                              <LockOutlined className={classes.inputIconsColor} />
-                            </InputAdornment>
-                          )
-                        }}
-                      />
-                    </CardBody>
-                    <CardFooter className={classes.cardFooter}>
-                      <Button simple color="primary" size="lg">
-                        Register
-                      </Button>
-                    </CardFooter>
-                  </form>
-                </Card>
-              </GridItem>
-            </GridContainer>
-          </div>
+                      labelText="Confirm Password"
+                      id="regPasswordConfirm"
+                      name="regPasswordConfirm"
+                      onChange={this.handleChange}
+                      formHelperText={
+                        this.state.submitted && !this.state.regPasswordConfirm
+                          ? "Confirm password is required"
```

Comment:
Show message for if confirm password doesn't match password?

Reply:
Fixed in 2007ca7390ac52aa17413dabbc89b5aacbc91e9f